netty/shaded/src/testShadow/java/io/grpc/netty/shaded/ShadingTest.java

```diff
@@ -67,7 +68,7 @@ public void noNormalNetty() throws Exception {
   @Test
   public void serviceLoaderFindsNetty() throws Exception {
     assertThat(ServerBuilder.forPort(0)).isInstanceOf(NettyServerBuilder.class);
-    assertThat(ManagedChannelBuilder.forAddress("localhost", 1234))
+    assertThat(Grpc.newChannelBuilder("localhost:1234", InsecureChannelCredentials.create()))
```

Comment:
`TlsChannelCredentials.create()` also works here and is equivalent to the old code.

Reply:
okay